lib/srv/db/common/auth.go

```diff
@@ -645,30 +703,30 @@ func (a *dbAuth) GetAzureCacheForRedisToken(ctx context.Context, sessionCtx *Ses
 // For RDS/Aurora, the config must contain RDS root certificate as a trusted
 // authority. For on-prem we generate a client certificate signed by the host
 // CA used to authenticate.
-func (a *dbAuth) GetTLSConfig(ctx context.Context, sessionCtx *Session) (*tls.Config, error) {
-	dbTLSConfig := sessionCtx.Database.GetTLS()
+func (a *dbAuth) GetTLSConfig(ctx context.Context, expiry time.Time, database types.Database, databaseUser string, databaseName string) (*tls.Config, error) {
```

Comment:
same as above. do we need `databaseName` for `GetTLSConfig`?

Reply:
Nope, good catch.